src/lib/tradingView/dydxfeed/index.ts

```diff
@@ -96,6 +116,41 @@ export const getDydxDatafeed = (
     setTimeout(() => onSymbolResolvedCallback(symbolInfo), 0);
   },
 
+  getMarks: async (
+    symbolInfo: LibrarySymbolInfo,
+    from: number,
```

Comment:
can we rename these `fromSeconds` and `toSeconds` or some other name that describes denomination